config.json

```diff
@@ -1,14 +1,18 @@
 {
-    "Mador" : {
-        "Keva" : {
-            "Harel Finaly" : ["",""],
-            "Raz Gino" : ["",""]
-        },
-        "Sadir" : {
-            "Ofek Shmuel": ["Ofek.Shmuel1@gmail.com", "0543238582" ],
-            "Ofek Ophir" : ["",""],
-            "Koren Minchev" : ["",""],
-            "Alon Alexander" : ["",""]
-        }
+    "Keva" : {
+        "Harel Finaly" : ["test@gmail.com",""],
+        "Raz Gino" : ["test@gmail.com",""],
+        "Dudu Eliyahu": ["test@gmail.com",""],
+        "Almog Kerdstein": ["test@gmail.com",""],
+        "Noam Barkay" : ["test@gmail.com",""]
+    },
+    "Sadir" : {
+        "Ofek Shmuel": ["Ofek.Shmuel1@gmail.com", "0543238582"],
```

Comment:
make sure the repo is private so that not everyone has access to phone numbers.

Reply:
fck, i got it on gitignore... what happend